pyphare/pyphare/core/gridlayout.py

```diff
@@ -229,7 +229,8 @@ def particleGhostNbr(self, interp_order):
 
     def nbrGhosts(self, interpOrder, centering):
         if self.field_ghosts_nbr == -1:
-            return int((interpOrder + 1) / 2) + self.particleGhostNbr(interpOrder)
+            nGhosts = int((interpOrder + 1) / 2) + self.particleGhostNbr(interpOrder)
```

Comment:
I think this function would deserve a very well written docstring that explains why this choice of number of ghosts because this starts to be a bit cryptic...